server/game/provincecard.js

```diff
@@ -138,6 +145,11 @@ class ProvinceCard extends BaseCard {
             !this.isBroken
         );
     }
+
+    hasEminent() {
+        //Facedown provinces are out of play and their effects don't evaluate, so we check for the printed keyword
+        return this.hasKeyword('eminent') || (!this.isBlank() && this.hasPrintedKeyword('eminent'));
```

Comment:
We may need to check whether blanking persists on provinces turned Face-Down to make sure blank + flip interaction works correctly.

Reply:
"If you blank an eminent province and then flip it facedown, it will remain facedown until it is attacked (or turned faceup by a card effect)."  from Tyler

Reply:
Awesome. This is fine too then.